admin/class-boldgrid-backup-admin-cli.php

```diff
@@ -50,7 +50,8 @@ public static function get_execution_functions() {
 		}
 
 		// Get the PHP disable_functions list.
-		$disabled = explode( ',', ini_get( 'disable_functions' ) );
+		$disabled = str_replace( ' ', '', ini_get( 'disable_functions' ) );
```

Comment:
You should remove tabs as well.  There could also be returns and line breaks.  Why not use pre_replace or array_walk with trim (which trims all of the whitespace chars).

```
$ php -r '$s = "one, two,\tthree"; var_dump( $s ); var_dump( preg_replace( "/[ \n\r\t\v]/", "", $s ) );'
string(15) "one, two,   three"
string(13) "one,two,three"
```

```
$ php -r '$s = "one, two,\tthree"; var_dump( $s ); function trim_val( &$v ) { $v = trim( $v ); } $a = explode( ",", $s ); array_walk( $a, "trim_val" ); var_dump( $a );'
string(15) "one, two,   three"
array(3) {
  [0]=>
  string(3) "one"
  [1]=>
  string(3) "two"
  [2]=>
  string(5) "three"
}
```

Reply:
Updated in https://github.com/BoldGrid/boldgrid-backup/pull/450/commits/d73e9d056c1e48875860afac369933b09b92e34d